add.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+require('functions.php');
+
+$db = new PDO('mysql:host=db; dbname=project2-paintings', 'root', 'password');
```

Comment:
move this into an external file that can be gitignored - that will keep credentials private

Reply:
also make it a function that returns a db connection